web/packages/teleport/src/DesktopSession/useTdpClientCanvas.tsx

```diff
@@ -282,6 +272,11 @@ export default function useTdpClientCanvas(props: Props) {
   // on the remote machine.
   const canvasOnContextMenu = () => false;
 
+  const windowOnResize = throttle((cli: TdpClient) => {
```

Comment:
I think `debounce` would be a better choice than `throttle` here.

https://web.archive.org/web/20220714020647/https://bencentra.com/code/2015/02/27/optimizing-window-resize.html

> debouncing window.onresize will only call the event handler after the event has stopped firing for a certain amount of time. This will ensure that your function will only be called once the resizing is “complete.”